types/node-telegram-bot-api/index.d.ts

```diff
@@ -1386,6 +1403,61 @@ declare class TelegramBot extends EventEmitter {
         fileOptions?: TelegramBot.FileOptions,
     ): Promise<TelegramBot.Message>;
 
+    getStickerSet(
+        name: string,
+        options?: {},
+    ): Promise<TelegramBot.StickerSet>;
+
+    getCustomEmojiStickers(
+        customEmojiIds: string[],
+        options?: {},
+    ): Promise<TelegramBot.Sticker[]>;
+
+    uploadStickerFile(
+        userId: string,
+        pngSticker: string | Stream | Buffer,
+        options?: {},
+        fileOptions?: TelegramBot.FileOptions,
+    ): Promise<TelegramBot.File>;
+
+    createNewStickerSet(
+        userId: string,
+        name: string,
+        title: string,
+        emojis: string,
+        pngSticker?: string | Stream | Buffer,
```

Comment:
Is order of parameters `pngSticker` and `title` wrong? According to [doc](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#telegrambotcreatenewstickersetuserid-name-title-pngsticker-emojis-options-fileoptions--promise) and [implementation](https://github.com/yagop/node-telegram-bot-api/blob/ccdd14670b10548eda32397cff3f3ed678991f0a/src/telegram.js#L2221), the parameter `pngSticker` is before `emojis`.